src/routes/auth.js

```diff
@@ -1,9 +1,10 @@
 import express from 'express';
+
 import { signup, signin } from '../controllers/auth';
+import signupValidator from '../middlewares/AuthValidator';
```

Comment:
We will need a validator middleware that will interface with the different validators for each route. Please look into this.